php/5.6/Dockerfile

```diff
@@ -0,0 +1,96 @@
+FROM php:5.6-fpm
+
+LABEL maintainer="Abhijit Rakas <abhijitrakas@gmail.com>"
+LABEL org.label-schema.schema-version="1.0.0-rc1"
+LABEL org.label-schema.vendor="EasyEngine"
+LABEL org.label-schema.name="php"
+
+RUN set -ex; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends \
+		imagemagick \
+		less \
+		libc-client-dev \
+		libfreetype6-dev \
+		libjpeg-dev \
+		libjpeg62-turbo-dev \
+		libkrb5-dev \
+		libmagickwand-dev \
+		libmcrypt-dev \
+		libmemcached-dev \
+		libxml2-dev \
+		libpng-dev \
+		libzip-dev \
+		mysql-client \
+		ssmtp \
+		unzip \
+		vim \
+		zip
+
+RUN pecl install imagick memcached-2.2.0; \
+	printf "\n" | pecl install mcrypt-1.0.1; \
+	printf "\n" | pecl install redis; \
+	docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr; \
+	docker-php-ext-configure zip --with-libzip; \
+	docker-php-ext-configure --with-php-config=/usr/local/bin/php-config; \
+	docker-php-ext-configure imap --with-kerberos --with-imap-ssl; \
+	docker-php-ext-install gd imap mysqli opcache soap zip; \
+	docker-php-ext-enable imagick mcrypt redis; \
+	echo "extension=memcached.so" >> /usr/local/etc/php/conf.d/memcached.ini; \
+	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
+	rm -rf /var/lib/apt/lists/*;
+
+RUN	mkdir -p /usr/src/php/ext; \
+	cd /usr/src/php/ext/; \
+	curl -sSL -o php7.zip https://github.com/websupport-sk/pecl-memcache/archive/NON_BLOCKING_IO_php7.zip; \
```

Comment:
Is this the correct version of memcache that we're using? The URL itself contains `php7`

Reply:
@mbtamuli URL contains `php7` because of `branch` name. No official words in `readme` about php5.6 support but repo contains code for both PHP version `5` and `7` though need to test once.

Reply:
@sagarnasit Did you get a chance to test this?

Reply:
@sagarnasit Please add yourself also as maintainer.